winterize/mod.conf

```diff
@@ -0,0 +1,2 @@
+name = winterize
+depends = default, ctf_map, wool
```

Comment:
Hmm, is `wool` a mistake? Or is there more coming? :D

Reply:
I was going too but then I thought it might ruin the theme of it. Shall I make a slab version of default:snow?

Reply:
I was going to but then I thought it might ruin the theme of it. Shall I make a slab version of default:snow?

Reply:
Otherwise I'll remove that

Reply:
Maybe we can leave wool alone, idk really...

> Shall I make a slab version of default:snow?

I don't understand. Don't we already have `default:snow` which is a slab? Or are you referring to the half-node kind of slab. If so, why do we need it?

Reply:
Regular snow in an all-wool map looks weird

Reply:
I'll just skip this. Removing dependency...